crates/core/src/subscription/module_subscription_actor.rs

```diff
@@ -21,73 +21,56 @@ use spacetimedb_lib::Identity;
 use tokio::sync::mpsc;
 
 #[derive(Debug)]
-enum ModuleSubscriptionCommand {
+enum Command {
```

Comment:
I think a comment on `Command` (or on the mpsc queue of them) describing the ordering invariants we have to maintain would be appropriate.

Reply:
There's not really any ordering invariants, they were just getting processed out-of-order of being sent because I had put 2 queues for some reason. Like, if you did `for _ in 0..100 { broadcast_commit_event(); }; update_subscription()` the actor could process the subscription update before getting through the whole backlog of commit events.

Reply:
Yes, the fact that they have to be one queue is exactly what you should describe in a comment in this code please.

Reply:
Thanks!